bin/test/i18n/resources/pegasus/emails/test_sync_in.rb

```diff
@@ -0,0 +1,47 @@
+require_relative '../../../../test_helper'
+require_relative '../../../../../i18n/resources/pegasus/emails/sync_in'
+
+describe I18n::Resources::Pegasus::Emails::SyncIn do
+  def around
+    FakeFS.with_fresh {yield}
+  end
+
+  before do
+    STDOUT.stubs(:print)
+    I18n::Utils::PegasusMarkdown.stubs(:sanitize_file_header)
+  end
+
+  describe '.perform' do
+    it 'calls #process' do
+      I18n::Resources::Pegasus::Emails::SyncIn.any_instance.expects(:process).once
+
+      I18n::Resources::Pegasus::Emails::SyncIn.perform
+    end
+  end
+
+  describe '#process' do
+    let(:sync_in) {I18n::Resources::Pegasus::Emails::SyncIn.new}
+
+    before do
+      FileUtils.mkdir_p(File.dirname(origin_emails_file_path))
+      FileUtils.touch(origin_emails_file_path)
+    end
+
+    context 'when the origin email file is .md' do
+      let(:origin_emails_file_subpath) {'public/test.md'}
```

Comment:
I don't think we need `public` in the path:
```suggestion
      let(:origin_emails_file_subpath) {'test.md'}
```